tests/bootstrap.php

```diff
@@ -21,6 +21,9 @@
 require $test_root . '/includes/functions.php';
 
 function _manually_load_plugin() {
+	// Make sure the plugin is "enabled"
+	define( 'S3_UPLOADS_AUTOENABLE', true );
```

Comment:
From what I understand, this is not a required constant, in which case I would rather run the unit tests without it defined as per the majority use case, and lets us write unit tests specifically with it defined.